src/models/signature.ts

```diff
@@ -5,6 +5,15 @@ export interface SignatureOptions {
   sig: string;
 }
 
+/**
+ * A container class containing information about a signature.
+ *
+ * Contains a signature and the keyid uniquely identifying the key used
+ * to generate the signature.
+ *
+ * Provides utility methods to easily create an object from a dictionary
+ * and return the dictionary representation of the object.
```

Comment:
Should this just say something about the `fromJSON` method? 
```suggestion
 *
 * Provide a `fromJSON` method to create a Signature from a JSON object.
```